config/puma.rb

```diff
@@ -43,7 +43,7 @@
 # Allow puma to be restarted by `rails restart` command.
 plugin :tmp_restart
 
-after_worker_boot do
-  require 'prometheus_exporter/instrumentation'
-  PrometheusExporter::Instrumentation::Puma.start(type: 'puma')
-end
+# Metrics
+activate_control_app
```

Comment:
Can you elaborate on implications of this call? I know that they mention it in the Usage, but what it actually does?

Reply:
It exposes the control of puma via a local UNIX socket, it is required for the extraction of metrics and setting up the metrics server as well.